base/tks/src/main/java/org/dogtagpki/server/tks/servlet/SecureChannelProtocol.java

```diff
@@ -1023,8 +1040,8 @@ public SymmetricKey unwrapSymKeyOnToken(CryptoToken token, SymmetricKey unwrappi
 
             // SecureChannelProtocol.debugByteArray(wrappedKey, " encrypted key");
 
-            KeyWrapper keyWrap = token.getKeyWrapper(KeyWrapAlgorithm.DES3_ECB);
-            keyWrap.initUnwrap(encUnwrapKey, null);
```

Comment:
ok I think this answered my previous question. ;-)

Reply:
If only you had put this one first :)